server/service/apple_mdm.go

```diff
@@ -2737,13 +2737,18 @@ func (svc *MDMAppleCheckinAndCommandService) CommandAndReportResults(r *mdm.Requ
 		host.Hostname = deviceName
 		host.GigsDiskSpaceAvailable = availableDeviceCapacity
 		host.GigsTotalDiskSpace = deviceCapacity
-		var osVersionPrefix string
+		var (
+			osVersionPrefix string
+			platform        string
+		)
 		if strings.HasPrefix(productName, "iPhone") {
-			osVersionPrefix = "iOS "
+			osVersionPrefix = "iOS"
+			platform = "ios"
 		} else { // iPad
-			osVersionPrefix = "iPadOS "
+			osVersionPrefix = "iPadOS"
+			platform = "ipados"
 		}
-		host.OSVersion = osVersionPrefix + osVersion
+		host.OSVersion = osVersionPrefix + " " + osVersion
```

Comment:
Why do we need the prefix here? Is it just for consistency with how osquery reports it?

Reply:
It's how we set the `hosts.os_version`, OS name + OS version, e.g.:
```
select os_version from hosts;
+---------------+
| os_version    |
+---------------+
| iPadOS 17.5.1 |
| iOS 17.5.1    |
| macOS 14.5    |
+---------------+
```